README-deploy-benchmark.md

```diff
@@ -0,0 +1,221 @@
+# ChatQnA Benchmarking
+
+This folder contains a collection of Kubernetes manifest files for deploying the ChatQnA service across scalable nodes. It includes a comprehensive [benchmarking tool](https://github.com/opea-project/GenAIEval/blob/main/evals/benchmark/README.md) that enables throughput analysis to assess inference performance.
```

Comment:
The GenAIExamples folder contains a lot of things .. so lets not get into discussing its organization. Just get right into benchmarking related material.

OPEA benchmarking supports both performance and accuracy testing. It leverages scripts in https://github.com/opea-project/GenAIEval/blob/main/evals/benchmark/README.md.